lib/balance-tx/lib/Cardano/Write/Tx/SizeEstimation.hs

```diff
@@ -110,7 +108,7 @@ import qualified Cardano.Wallet.Primitive.Types.Address as W
 import qualified Cardano.Wallet.Primitive.Types.Coin as Coin
 import qualified Cardano.Wallet.Primitive.Types.TokenBundle as TokenBundle
 import qualified Cardano.Wallet.Primitive.Types.Tx.TxOut as W
-import qualified Cardano.Wallet.Shelley.Compatibility.Ledger as W
```

Comment:
On master:
```bash
$ git grep -o "W\.[a-zA-Z]*" | grep "Cardano.Write.Tx" | grep -o "W\.[a-zA-Z]*" | sort -u
W.Address
W.AssetId
W.Coin
W.Hash
W.ProtocolParameters
W.TokenBundle
W.TokenMap
W.TokenPolicyId
W.TokenQuantity
W.TxIn
W.TxOut
W.UTxO
W.UnsafeTokenPolicyId
W.fromBabbageTxOut
W.fromConwayTxOut
W.toBabbageTxOut
W.toConwayTxOut
W.toLedger
W.toLedgerAddress
W.toLedgerCoin
W.toLedgerTimelockScript
W.toWallet
W.toWalletAddress
W.toWalletCoin
W.toWalletTokenBundle
```

What do you think about importing `Compatibility.Ledger as W` instead of `as Convert`?

My idea was to use the `W` prefix for a similar purpose to your `Convert` in the `Write.Tx` hierarchy. As you noticed, there were cases of unqualified `Compatibility.Ledger` imports. But I don't see why we'd want to grep for `W/Convert.toBabbageTxOut` separately from `W.TxOut`. Both are due for removal.

Reply:
> What do you think about importing Compatibility.Ledger as W instead of as Convert?

I'm not too keen on this, because the `W` namespace is already used for other things, and I'm trying here to give conversions their own namespace that is distinct from wallet primitive types.